linkml/generators/common/__init__.py

```diff
@@ -0,0 +1,17 @@
+from linkml_runtime.linkml_model.meta import SlotDefinition, ClassDefinition
+from linkml_runtime.utils.schemaview import SchemaView
+from linkml_runtime.utils.formatutils import camelcase
+
+def get_type_designator_value(sv: SchemaView, type_designator_slot: SlotDefinition, class_def: ClassDefinition) -> str:
```

Comment:
is the __init__.py the best place for this?

Reply:
shall i put it in a separate `type_designators.py` under `linkml/generators/common` ? can i edit it directly in this branch ?

Reply:
`linkml/generators/common/type_designators.py` sounds like a good place - I think you should be able to edit the branch, let us know if you can't.